examples/gcoap/gcoap_cli.c

```diff
@@ -30,6 +30,27 @@
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+#if IS_ACTIVE(CONFIG_GCOAP_ENABLE_DTLS)
+#include "net/credman.h"
+#include "net/dsm.h"
+#include "tinydtls_keys.h"
+
+#define GCOAP_DTLS_CREDENTIAL_TAG 10
```

Comment:
None. It's just a number I used here for the example. Credman does not allow storing multiple credentials with the same tag + type (PSK/RPK). The user would need to avoid same tag numbers in real applications.

Reply:
better add a comment, since this will serve as an example